bin/create-maintenance-release

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+set -e
+
+# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - FUNCTIONS
+# require helper
+currentDir=$(node -e "console.log(require('path').dirname(require('fs').realpathSync('$BASH_SOURCE')))")
+source $currentDir/helper
+
+help () {
+  log ""
+  log "This script creates a maintenance release (for the upstream e.g. livingdocs-editor / livingdocs-server."
+  log ""
+  log "-----------------------------------------------------------------------------------------------------------------------"
+  log "Parameters"
+  log "-----------------------------------------------------------------------------------------------------------------------"
+  log "--help / -h               Show help"
+  log "--base-tag                Base tag where the maintenance branch will be branched from e.g. '--base-tag v55.0.0'"
+  log "--npm-auth-token          npm-auth-token with write access e.g. '--npm-auth-token 11111111-2222-3333-4444-555555555555'"
+  log "--dry-run                 No execution of the script (just logs) '--dry-run true' (default: false)"
+  log ""
+  exit 1
+}
+
+assignCLIParameters () {
+  # parse command line argumens in bash
+  # https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash
+  while [[ $# -gt 0 ]]
```

Comment:
But yes, when we would consequently rewrite all scripts to node... maybe it would make sense.